recipes/libmicrohttpd/all/conanfile.py

```diff
@@ -171,9 +172,14 @@ def build(self):
     def package(self):
         copy(self, "COPYING", os.path.join(self.source_folder), os.path.join(self.package_folder, "licenses"))
         if is_msvc(self):
-            copy(self, "*.lib", os.path.join(self.build_folder, self._msvc_sln_folder, "Output", self._msvc_arch), os.path.join(self.package_folder, "lib"))
-            copy(self, "*.dll", os.path.join(self.build_folder, self._msvc_sln_folder, "Output", self._msvc_arch), os.path.join(self.package_folder, "bin"))
-            copy(self, "*.h", os.path.join(self.build_folder, self._msvc_sln_folder, "Output", self._msvc_arch), os.path.join(self.package_folder, "include"))
+            # 32-bit (x86) libraries are stored in the root
+            copy(self, "*.lib", os.path.join(self.build_folder, self._msvc_sln_folder, "Output"), os.path.join(self.package_folder, "lib"))
```

Comment:
Can you use a variable to simplify all of this a lot?

Reply:
See [cf2db17](https://github.com/conan-io/conan-center-index/pull/13699/commits/cf2db17ee28894233768b627031d460c27d21169).
More lines of code, but more understandable.